app/forms/search_developer_form.rb

```diff
@@ -9,16 +9,24 @@ class SearchDeveloperForm
 
   def search
     query = Developer.all
-    return query if programming_language_id.blank? && language_id.blank?
+    return query if !programming_language_id? && !language_id?
```

Comment:
What I meant was that the line looks unnecessary.
Because the conditions are same as those in line 14 and 15.
So it just makes other developers read extra logic.

Reply:
@jp-ryuji (Y)